gallery/src/styles/NxFormLayout/NxFormLayoutExample.tsx

```diff
@@ -70,6 +72,9 @@ export default function NxFormLayoutExample() {
 
   return (
     <form className="nx-form" onSubmit={onSubmit} aria-label="Default Form Layout Example">
+      <NxTile.HeaderSubtitle>
```

Comment:
Another similar to https://github.com/sonatype/sonatype-react-shared-components/pull/735#discussion_r914795669

Reply:
Updated in [cd9d67e](https://github.com/sonatype/sonatype-react-shared-components/pull/735/commits/cd9d67e06c9a7d3c9eba14c9f8d1c407da96c88c)